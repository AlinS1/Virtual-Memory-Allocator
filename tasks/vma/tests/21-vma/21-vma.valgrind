==2905== Memcheck, a memory error detector
==2905== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==2905== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==2905== Command: /mnt/c/Users/alins/OneDrive/Desktop/SDA_tema1/vma
==2905== Parent PID: 2904
==2905== 
==2905== 
==2905== HEAP SUMMARY:
==2905==     in use at exit: 0 bytes in 0 blocks
==2905==   total heap usage: 44 allocs, 44 frees, 2,040 bytes allocated
==2905== 
==2905== All heap blocks were freed -- no leaks are possible
==2905== 
==2905== For lists of detected and suppressed errors, rerun with: -s
==2905== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
