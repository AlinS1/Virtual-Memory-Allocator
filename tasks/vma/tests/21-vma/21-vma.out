Total memory: 0x10000 bytes
Free memory: 0xFFE2 bytes
Number of allocated blocks: 3
Number of allocated miniblocks: 3

Block 1 begin
Zone: 0x1000 - 0x100A
Miniblock 1:		0x1000		-		0x100A		| RW-
Block 1 end

Block 2 begin
Zone: 0x3000 - 0x300A
Miniblock 1:		0x3000		-		0x300A		| RW-
Block 2 end

Block 3 begin
Zone: 0x3014 - 0x301E
Miniblock 1:		0x3014		-		0x301E		| RW-
Block 3 end
This zone was already allocated.
Total memory: 0x10000 bytes
Free memory: 0xFFE2 bytes
Number of allocated blocks: 3
Number of allocated miniblocks: 3

Block 1 begin
Zone: 0x1000 - 0x100A
Miniblock 1:		0x1000		-		0x100A		| RW-
Block 1 end

Block 2 begin
Zone: 0x3000 - 0x300A
Miniblock 1:		0x3000		-		0x300A		| RW-
Block 2 end

Block 3 begin
Zone: 0x3014 - 0x301E
Miniblock 1:		0x3014		-		0x301E		| RW-
Block 3 end
Invalid address for free.
Total memory: 0x10000 bytes
Free memory: 0xFFE2 bytes
Number of allocated blocks: 3
Number of allocated miniblocks: 3

Block 1 begin
Zone: 0x1000 - 0x100A
Miniblock 1:		0x1000		-		0x100A		| RW-
Block 1 end

Block 2 begin
Zone: 0x3000 - 0x300A
Miniblock 1:		0x3000		-		0x300A		| RW-
Block 2 end

Block 3 begin
Zone: 0x3014 - 0x301E
Miniblock 1:		0x3014		-		0x301E		| RW-
Block 3 end
